templates/portal/home.html

```diff
@@ -174,12 +174,63 @@ <h3>Collections
     </div>
     {% endif %}
 
+    {% if events_objects %}
+    <div class="row text-center mt-5">
+        <div class="col-12 text-center">
+            <h3>Events</h3>
```

Comment:
```suggestion
            <h3><a href="{% url evens %}">Events</a></h3>
```

I'd think it should be like this.

Reply:
Great point, and thanks to you I now found a mistake in the views.py, thanks! Solved this differently.